actions/testmo-create-resources/action.yml

```diff
@@ -0,0 +1,38 @@
+name: create testmo resources
+description: 'create a TestMo resource .json file'
+inputs:
+  resources_file:
+    description: "file name where the resources will be stored"
+    required: true
+  python_version:
+    description: "version of python"
+    default: ""
+    required: false
+  runner:
+    description: "runner hardware configuration label"
+    default: ""
+    required: false
+  lm_eval_configuration:
+    description: "list of configurations tested with LM-Eval"
+    default: ""
+    required: false
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        echo "composing TESTMO resources file ${{ inputs.resources_file }} ..."
+        declare -A fields=( \
+          ["python"]="${{inputs.python_version}}" \
+          ["runner"]="${{inputs.runner}}" \
+          ["lm_eval_configuration"]="${{inputs.lm_eval_configuration}}" \
+        )
+        for i in "${!fields[@]}"; do 
+          if [[ "${fields[$i]}" ]]; then
+            npx testmo automation:resources:add_field \
+              --resources "${{ inputs.resources_file }}" \
```

Comment:
This should be for the --thread-resources not for the whole testmo run resources right?

Reply:
this `npx testmo automation:resources:add_field` command is adding a field to a JSON file that will be passed to the `npx testmo automation:run:submit-thread` command via the `testmo-run-submit-thread/action.yml` file in the actions repo.  
The line here is pointing to the JSON file where the resources are to be stored.

Reply:
I meant the resources should be for the --thread-resources as we're submitting results from Test and lm-eval jobs which has its own resource contents.

Reply:
Well, the resources created with this action could be used for any `npx testmo automation:run:*` command.  for my nm-vllm-ent repo changes, I'll specifically be using the file w/ the TestMo run's thread, but that's not really relevant at this line in the code.

some links to the docs might help?

https://docs.testmo.com/docs/automation/reference#submitting-threads-for-runs
https://docs.testmo.com/docs/automation/reference#adding-fields-links-and-artifacts

Reply:
The fields we need to add are only for threads not for the entire testmo run. As mentioned in Domenic's comment as well, we just need to add these fields info for the --thread-resources argument.

Reply:
This command has no relation to threads or runs, though, it is simply building a resources file (which itself is independent of threads or runs). This command to build the file only has a `--resources` flag because those concepts are unrelated to building the resources, they are only related to actually submitting them to a thread/run.

Reply:
Yup, that's being done in the other file in this PR.  The command in *this* file is to add fields to a resources file whose path is provided by the `--resources` arg on this line.

Reply:
But those fields should only apply to thread right?

Reply:
Dan and I discussed this after standup.  I believe that it's now clear that this `--resources` arg is simply the name of the resources file, and how the same file name is passed to the `--thread_resources` arg for testmo's `automation:run:submit-thread` command.  Marking this convo resolved.